apps/src/lib/kits/maker/util/makeStubBoard.js

```diff
@@ -1,7 +1,7 @@
 import five from '@code-dot-org/johnny-five';
 import Playground from 'playground-io';
 
-export function makeStubBoard() {
+export function CPBoardStub() {
```

Comment:
nit: might be good to retain the `make` verbiage as this is a function that returns a board (rather than an implementation of Board class)? maybe `makeCPBoardStub` or `makeStubCPBoard`?

Reply:
Oh! That makes sense. Will update to `makeCPBoardStub`. Thanks!